files/rack_attack.rb

```diff
@@ -0,0 +1,29 @@
+# 1. Rename this file to rack_attack.rb
```

Comment:
this file is not ansible managed

Reply:
it is if you copy it in this role.
`ansible_managed` means that nobody should change the file manually, because the changes will be lost on the next playbook run.